Specs/DateTime/English/DateExtractor.json

```diff
@@ -1587,5 +1587,11 @@
         "Length": 26
       }
     ]
+  },
+  {
+    "Input": "6,107.31 August 2019 should not include the decimal",
+    "Comment": "Something wrong with negative lookbehind in java",
```

Comment:
Change this comment to: "Only August 2019 should be extracted as a DateRange, so no output in Date only. Java disabled due to issue in lookbehind."